Theory of Computing.js

```diff
@@ -67,127 +65,119 @@
  ******/
 
 
-
-
-//A Regexp for extracting the Domain
+// A Regexp for extracting the Domain
 
 var surveyRegexp = new RegExp("articles/(gs[^/]+)");
 var journalRegexp = new RegExp("articles/(v[^/]+)");
 
-function detectWeb(doc, url){
-	if (surveyRegexp.test(url))
-		return "book";
-	else
-		return "journalArticle";
+function detectWeb(doc, url) {
+	if (surveyRegexp.test(url)) return "book";
+	else return "journalArticle";
 }
 
-function doWeb(doc, url){
+function doWeb(doc, url) {
 	var typeRegExp;
 	var type = detectWeb(doc, url);
 
-	//Select the right regexp according to the article type
-	if (type == "journalArticle")
-		typeRegExp = journalRegexp;
-	else
-		typeRegExp = surveyRegexp;
+	// Select the right regexp according to the article type
+	if (type == "journalArticle") typeRegExp = journalRegexp;
+	else typeRegExp = surveyRegexp;
 
 	var urlData = typeRegExp.exec(url);
-	//urlData is an Array of three elements, with the element at index 1 containg the
-	//base domain of the mirror being accesses, and the element at index 2
-	//containing the article ID.
+	// urlData is an Array of three elements, with the element at index 1 containg the
+	// base domain of the mirror being accesses, and the element at index 2
+	// containing the article ID.
 
 	var articleID = urlData[1];
 
-	//We now start constructing the Zoetro item
+	// We now start constructing the Zoetro item
 	var newItem = new Zotero.Item(type);
 
-	//Store the snapshot and the PDF file
-	newItem.attachments.push({
-		title: "Theory of Computing Snapshot",
-		document: doc});
+	// Store the snapshot and the PDF file
+	newItem.attachments.push({ title: "Theory of Computing Snapshot",
+		document: doc });
```

Comment:
Which rule is this? This looks unfamiliar for me...

Reply:
There's a rule that allows such indenting, but not one that enforces it. If I put it on one line, it stays there. Alternate is to only indent the keys (which I think looks best, and which I've just submitted).

Reply:
Agree, I like this option also better.